### PR TITLE
Add support for dependency groups

### PR DESCRIPTION
**Pull request by tofay**
By making `BundleCommand` inherit from poetry's `GroupCommand`, and passing the detected groups through to the venv bundler.

Fixes https://github.com/python-poetry/poetry-plugin-bundle/issues/8

### CODE REVIEW

**Comment by SamambaMan** [comment]
Hey! 

I need this SO BADLY and would be awesome if the PR were accepted.
Any chances? :D

Thanks for the great work

**Comment by connelldave** [comment]
Hello - ➕ this solves a blocking problem for my project. Curious if this is a change you're open to accepting? Thanks!

**Comment by mxab** [comment]
This looks really helpful!
@neersighted would it be possible to ping the reviewers, would be a shame if that PR goes to waste :)

File: tests/bundlers/test_venv_bundler.py [hunk 1]
```diff
@@ -221,3 +221,51 @@ def test_bundler_should_display_a_warning_for_projects_with_no_module(
...
+    poetry.set_pool(pool)
+
+    mocker.patch("poetry.installation.executor.Executor._execute_operation")
+
+    bundler = VenvBundler()
+    bundler.set_path(Path(tmpdir))
+    bundler.set_remove(True)
+
+    # This should fail because there is not repo for bar
+    from poetry.puzzle.exceptions import SolverProblemError
```
Comment (radoering) [inline tests/bundlers/test_venv_bundler.py:250]: This import should be global.

**Review by radoering — CHANGES_REQUESTED** [review]
Sorry for the late feedback. Overall this looks good to me. Can you add some simple tests with the different options in `tests/console/commands/bundle/test_venv.py`, please?

**Comment by tofay** [comment]
I've added the tests as suggested.

**Review by radoering — APPROVED** [review]

**Pull request merged**